buildtools/fsyacc/fsyaccast.fs

```diff
@@ -38,7 +38,7 @@ type Symbols = Symbol list
 //---------------------------------------------------------------------
 // Output Raw Parser Spec AST
 
-let StringOfSym sym = match sym with Terminal s -> "'" ^ s ^ "'" | NonTerminal s -> s
+let StringOfSym sym = match sym with Terminal s -> String.Concat("'", s, "'") | NonTerminal s -> s
```

Comment:
Can you please do a final "62" full text search?
This file and the fslex files still have it, among others.

I have also not seen a removal of the message behind "62", which I found strange.. ?

Reply:
This was removed https://github.com/dotnet/fsharp/blob/0b836fb40a5edbe12d32414c330c806a0655fb61/src/Compiler/Facilities/DiagnosticsLogger.fs#L613-L617

including fscomp texts